cmake/Modules/FindTPLTHREADS.cmake

```diff
@@ -11,7 +11,8 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(TPLTHREADS DEFAULT_MSG FOUND_THREADS)
 #Only create the TPL if we succeed
 IF (FOUND_THREADS)
   IF(USE_THREADS)
-    KOKKOS_CREATE_IMPORTED_TPL(THREADS INTERFACE LINK_LIBRARIES Threads::Threads)
+    KOKKOS_CREATE_IMPORTED_TPL(THREADS INTERFACE LINK_OPTIONS
+            ${TPL_Pthread_LIBRARIES})
   ELSE()
```

Comment:
Do we actually need to distinguish depending on `USE_THREADS`? Can't we use 
```
KOKKOS_CREATE_IMPORTED_TPL(THREADS INTERFACE LINK_OPTIONS
            ${TPL_Pthread_LIBRARIES})
```
in both cases?